### DISCUSSION TITLE
TTS button doesn't work on mobile

### DISCUSSION THREAD

**Discussion by colaber2** [open-webui/open-webui#2883] [General]
When I click the TTS button on my mobile, it keeps loading without producing any results. However, it works fine when I use it on my laptop.
![IMG_AB6F12762C4E-1](https://github.com/open-webui/open-webui/assets/153990548/2ca675d4-005f-4fc7-8e85-4e22d76f7d92)

**Comment by gkingt** [comment]
I also encountered this problem, the use of Android phone is fine, but iphone will not work properly. I tried to capture the packet and didn't seem to make any requests after clicking Play voice?
